server/db.js

```diff
@@ -0,0 +1,22 @@
+const {Client} = require('pg');
+
+const client = new Client({
+  database: 'megabites',
+  user: 'jane',
+});
+
+client.connect()
+  .then(() => { module.exports.client = client; })
+  .catch(err => console.log(err));
+
+const getRecipeTools = function (id) {
+  const query = {
+    text: 'SELECT tools.id, tools.name, tools.description, tools.manufacturer, tools.price, tools.imageURLs ' +

```

Comment:
rather than concatenating the two parts of the string using +, you can achieve the same result using backticks ` or separating lines using \